docs/features.md

```diff
@@ -30,6 +30,7 @@ GitLabForm enables you to manage the [(GitLab's) Application Settings](reference
     * [Push Rules](reference/push_rules.md) (**GitLab Premium (paid) only**)
     * [Integrations](reference/integrations.md)
     * [Settings](reference/settings.md#project-settings)
+
```

Comment:
Let's undo this change.

Reply:
Reopening this one. The latest diff as of now still shows this line.

I think it might split the feature list with a blank line. Since our CI pipeline doesn't build a review version for the website, I can't tell unless we manually verify it locally. So, might be safer to undo the change in this file.